playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.tsx

```diff
@@ -137,25 +192,24 @@ const PhoneNumberInput = (props: PhoneNumberInputProps) => {
 
   // Separating Concerns as React Docs Recommend
   // This also Fixes things for our react_component rendering on the Rails Side
-  useEffect(() => {
-    formatAllCountries()
-  }, [])
+  useEffect(formatAllCountries, [])
 
   useEffect(() => {
-    const telInputInit = new intlTelInput(inputRef.current, {
+    const telInputInit = intlTelInput(inputRef.current, {
       separateDialCode: true,
       preferredCountries,
       allowDropdown: !disabled,
+      autoInsertDialCode: false,
       initialCountry,
       onlyCountries,
-    }
-    )
+      utilsScript: "https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/18.1.6/js/utils.min.js"
```

Comment:
@gavhuang @thestephenmarshall Is this required in the useEffect? We separated it out into it's own file already, but i was curious, if we needed it here as well. 

Line 6 already brings it in: `import 'intl-tel-input/build/js/utils.js'`

Reply:
Nope. It can be removed as it is extraneous.

Reply:
My next PR #2613  removes this @jasperfurniss if you want to get this one in first.